source/ncconvert/converters/GeometryConverter.cpp

```diff
@@ -329,9 +332,34 @@ class GeometryConverter::impl
             };
         }
 
-        auto ImportMesh(const std::filesystem::path& path) -> asset::Mesh
+        auto ImportMesh(const std::filesystem::path& path, const std::optional<std::string>& internalName) -> asset::Mesh
         {
             const auto scene = ::ReadFbx(path, &m_importer, meshFlags);
+
+            aiMesh* mesh = nullptr;
+
+            if (internalName.has_value())
```

Comment:
Consider extracting the mesh selection code into a helper function.

Reply:
Not for now, just thinking about the future - I believe Assimp only stores that most reacent scene, but I think we could extract and cache them in the converter. Then whenever we go to load a mesh, we first check against the cache to see if we've already parsed that file.